lib/matplotlib/axes/_axes.py

```diff
@@ -1839,8 +1839,9 @@ def bar(self, *args, **kwargs):
             The x coordinates of the bars. See also *align* for the
             alignment of the bars to the coordinates.
 
-        height : scalar or sequence of scalars
+        height : scalar or array-like of shape(N,) or shape(2,N)
             The height(s) of the bars.
+            2-dimensional arrays represent grouped bars.
```

Comment:
I think this should be 

> scalar or array-like of shape(N,) or shape(N,M)

 right?  (same below for barh)
And the role of N and M should be explained here. Something like 

>..., where N is the number of bar positions (same as the length of `x`) and M the number of groups.